src/Controller/ArticlesController.php

```diff
@@ -956,13 +979,14 @@ private function articlePageArguments(Request $request, string $id, int $version
         return $arguments;
     }
 
-    private function createViewSelector(PromiseInterface $item, PromiseInterface $hasFigures, bool $isFiguresPage, PromiseInterface $history, PromiseInterface $sections) : PromiseInterface
+    private function createViewSelector(PromiseInterface $item, PromiseInterface $hasFigures, bool $isFiguresPage, PromiseInterface $history, PromiseInterface $sections, PromiseInterface $otherLinks) : PromiseInterface
```

Comment:
PHPUnit fails with a leftover call that still has 5 arguments?

Reply:
On the figures and data page.

Reply:
Should it be there? 🤔

Reply:
Assume the answer is yes. The DAR download will appear, but the banner itself won't...